symfony/client/src/pages.ts

```diff
@@ -50,6 +50,9 @@ import ViewOrganizationGeneralInformation from '@/orangehrmAdminPlugin/pages/org
 import Nationality from '@/orangehrmAdminPlugin/pages/nationality/Nationality.vue';
 import EditNationality from '@/orangehrmAdminPlugin/pages/nationality/EditNationality.vue';
 import SaveNationality from '@/orangehrmAdminPlugin/pages/nationality/SaveNationality.vue';
+import TerminationReason from '@/orangehrmPimPlugin/pages/configuration/TerminationReason.vue';
```

Comment:
`orangehrmPimPlugin` has index.ts files, you should import these files there. it will be automatically added here.